### DISCUSSION TITLE
BE节点storage_root_path配置多目录存储，数据却集中存储在一个目录

### DISCUSSION THREAD

**Discussion by jackchen863** [apache/doris#7059] [A - General / Q&A]
be节点be.conf配置存储如下
storage_root_path = /data1/doris14/be/storage;/data2/doris14/be/storage;/data3/doris14/be/storage;/data4/doris14/be/storage
最终数据集中往data1上写，直至磁盘使用达到95%，从而停止数据的load，并报以下错误
disk -3379323731653247649 on backend 10002 exceed limit usage
但是其它磁盘data2、data3、data4的使用率基本都在1%~2%左右
请问怎样使数据均衡存储，避免某个目录过多使用导致磁盘报警

**Comment by morningman** [comment]
后面的几个盘是后加上去的么？如果是后加上去的，只有新建的tablet才会到新的盘上。

**Reply by jackchen863** [reply]
不是后加的，安装的时候就这么配置的

**Reply by dataalive** [reply]
这个问题还在吗？
